sdk/batch/Microsoft.Azure.Batch/Tools/CodeGenerationLibrary/CodeGenerationLibrary.csproj

```diff
@@ -1,18 +1,17 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <PackageId>CodeGenerationLibrary</PackageId>
     <Description>Azure Batch object model code generation library</Description>
     <!--After fixing all warnings, enable this option-->
     <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
-    <AssemblyName>CodeGenerationLibrary</AssemblyName>
+    <AssemblyTitle>CodeGenerationLibrary</AssemblyTitle>
     <RestorePackagesPath>$(LibraryNugetPackageFolder)</RestorePackagesPath>
```

Comment:
@weshaggard Does this apply in for dataplane?

Reply:
I don't believe RestorePackagesPath applies.

Reply:
That said however I'm not entirely sure what it is being used for, it might be getting used by something other then nuget.